easybuild/easyblocks/generic/cmakemake.py

```diff
@@ -108,25 +125,23 @@ def configure_step(self, srcdir=None, builddir=None):
 
         options = ['-DCMAKE_INSTALL_PREFIX=%s' % self.installdir]
 
-        build_type = self.cfg['build_type']
-        if build_type is None:
-            build_type = 'Debug' if self.toolchain.options.get('debug', None) else 'Release'
-        options.append("-DCMAKE_BUILD_TYPE=%s" % build_type)
+        if 'DCMAKE_BUILD_TYPE' not in self.cfg['configopts']:
```

Comment:
I didn't use this because configopts overwrite these options. But if you want this can you use this to reduce false positives:
```suggestion
        if '-DCMAKE_BUILD_TYPE=' not in self.cfg['configopts']:
```

Reply:
it's useful to avoid having `-DCMAKE_BUILD_TYPE` twice in `configopts`, to avoid confusin

stricter pattern used in 05434817f